modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -93,14 +93,17 @@ test.describe('Manage object fields through Model Builder', () => {
 		modelBuilderPage,
 	}) => {
 		const {listTypeDefinitionIds, objectDefinition} = createdEntities;
-
-		const listTypeDefinitions = await Promise.all(
-			Array(22)
-				.fill(null)
-				.map(() =>
-					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
-				)
-		);
+				
+		let listTypeDefinitions = (await apiHelpers.listTypeAdmin.getListTypeDefinitions()).items;
+
+		listTypeDefinitions = listTypeDefinitions.concat(
+			await Promise.all(
+				Array(22)
+					.fill(null)
+					.map(() =>
+						apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+					)
+			));
```

Comment:
Hey @p-albuquerque, I know this code was here before, but we should be using an async `map` to make sense of using `Promise.all`, so we need to rewrite that map to make sure this async code is waiting correctly:

```js
await Promise.all(
  Array(22)
	  .fill(null)
	  .map(async () =>
		  await apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
	)
));
```